frontend/public/components/resource-pages.ts

```diff
@@ -278,6 +279,11 @@ export const baseDetailsPages = ImmutableMap<ResourceMapKey, ResourceMapValue>()
       (m) => m.PersistentVolumeClaimsDetailsPage,
     ),
   )
+  .set(referenceForModel(VolumeSnapshotModel), () =>
+    import('./pvc-snapshot' /* webpackChunkName: "snapshot" */).then(
```

Comment:
```suggestion
    import('./volume-snapshot' /* webpackChunkName: "volume-snapshot" */).then(
```

and make sure the folder name matches

Reply:
ack